src/pcapi/core/offers/validation.py

```diff
@@ -104,15 +104,15 @@ def check_stock_quantity(quantity: Union[int, None], bookingQuantity: int = 0) -
         raise api_errors
 
 
-def check_stock_price(price: float, offer_is_event: bool) -> None:
+def check_stock_price(price: float, offer_is_event: bool, offer_is_educational: bool) -> None:
```

Comment:
maintenant qu'on a 2 conditions sur l'offre, on pourrait peut-être passer l'offre directement en argument ?

```suggestion
def check_stock_price(price: float, offer: Offer) -> None:
```

Reply:
C'est fait
Mais à la réflexion, le paramètre `is_event` risque de sauter soon, pour permettre d'appliquer cette règle aux événements également (vu qu'en réalité, ils avaient voulu éviter d'impacter l'EAC avec ce flag initialement)